spec/amp-cache-guidelines.md

```diff
@@ -88,6 +88,13 @@ Accepted MIME types for fonts include the following prefixes:
 |  binary/octet-stream   |
 |  text/plain (not recommended)   |
 
+## Guidelines: Adding a new cache to the AMP ecosystem
+
+* Contact relevant publishers to make sure they update their CORS policies to include your origin.
```

Comment:
I know that this is the only way right now, but it obviously doesn't scale, which means UX will be bad :(